src/ui/Frame.cpp

```diff
@@ -1772,240 +1772,236 @@ void createTestUI() {
 }
 
 // sample function - would need to cache the blitted images somewhere for real-time use.
```

Comment:
got an idea of the performance penalty here? could put this functionality right in Image class - eg add some characters to the filename to cache an image with an outline. like `Image::get("myimage.png#0xffffffff#2")`

Reply:
If I made everything with an outline, the game drops from 140FPS shootmode/100FPS !shootmode to 100/60FPS in debug. Importantly the outline SDL surface is only created once and stored within the image, though each time it draws it needs to texture->bind() texture->load() like the minimap. Can refine it further if we ever have a use for it